reporter/index.js

```diff
@@ -9,7 +9,8 @@ const socket = io('http://localhost:5001/', {
     },
 });
 
-const events = require('./lib/events')(socket, id);
+const events = require('./lib/handlers')(socket, id);
+const runtimeEvents = require('./lib/runtimeEvents');
```

Comment:
It's just a constants file. If it's not used anywhere else, we can define the constants here itself and use them.